coursedata/level-defaults/nb_NO.yaml

```diff
@@ -1,354 +1,354 @@
+    example_code: "## Eksempelkode\n```\nnavn {is} Hedy\nalder {is} 15\n{print} navn is alder år gammel\n```\n"
+    intro_text_2: "`{print}` fungerer fortsatt som før, men `{ask}` kommandoen har endret seg. Fra nå må du bruke en variabel når du bruker `{ask}` kommandoen. Det ser slik ut:\n"
+    example_code_2: "```\nsvar {is} {ask} Hva heter du?\n{print} Hallo svar\n```\n"
+    intro_text_3: "Enda en ny kommando på dette nivået er `{sleep}`. Denne kommandoen setter programmet ditt på pause i et sekund. Hvis du skriver et tall bak {sleep} kommandoen, vil programmet settes på pause i så mange sekunder.\n"
+    example_code_3: "```\n{print} Min favorittfarge er...\n{sleep} 2\n{print} grønn!\n```\n"
+    start_code: '{print} hallo verden!'
     commands:
     -   name: '{is}'
-        explanation: Give a word a name to use in the program using `{is}`. You can choose the name yourself.
-        example: 'Example: name {is} Hedy.'
-        demo_code: "name {is} Hedy\n{print} welcome name"
+        explanation: Gi et ord et navn som kan brukes andre steder i programmet med kommandoen `{is}`. Du kan velge navnet helt selv.
+        example: 'Eksempel: navn {is} Hedy.'
+        demo_code: "navn {is} Hedy\n{print} Velkommen navn"
     -   name: '{ask}'
-        explanation: ask something with `{ask}`. Beware! You need to give the answer a name with `{is}`.
-        example: 'Example: color {is} {ask} What is your favorite color?'
-        demo_code: "color {is} {ask} What is your favorite color?\n{print} color is your favorite!"
+        explanation: still et spørsmål med `{ask}`. Pass på! Du må gi svaret et navn ved hjelp av `{is}`.
+        example: 'Eksempel: farge {is} {ask} Hva er din favorittfarge?'
+        demo_code: "farge {is} {ask} Hva er din favorittfarge?\n{print} farge er din favoritt!"
     -   name: '{sleep}'
-        explanation: Let Hedy pause for a (couple of) second(s).
-        example: 'Example: {sleep} 2 '
-        demo_code: "{print} Let me think for one second...\n{sleep}\n{print} Hmm.. I need 3 more seconds...\n{sleep} 3\n{print} Eureka! Ive got it!"
-    -   explanation: Give a number a name using `{is}`. You can choose the name yourself.
-        example: 'Example: angle {is} 90'
-        demo_code: "angle {is} 90\n{turn} angle\n{forward} 100"
-        name: is - turtle
+        explanation: La Hedy få seg en pause i et sekund eller fler.
+        example: 'Eksempel: {sleep} 2 '
+        demo_code: "{print} La meg tenke i et sekund...\n{sleep}\n{print} Hmm.. Jeg trenger tre sekunder til...\n{sleep} 3\n{print} Eureka! Jeg har svaret!"
+    -   explanation: Gi et tall et navn ved å bruke `{is}`. Du kan velge navnet selv.
+        example: 'Eksempel: vinkel {is} 90'
+        demo_code: "vinkel {is} 90\n{turn} vinkel \n{forward} 100"
```

Comment:
For some reason this crashes our tests. It might have something to do with the extra whitespace at `vinkel \n`. I fixed this on Weblate but will keep this as requested changes until it is fixed.

Reply:
This seems to be solved!